build-tools/libzip-windows/libzip-windows.targets

```diff
@@ -1,3 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Target 
+    Name="Build"
```

Comment:
This line -- the `Name` attribute -- should be on the previous line, because I like `<Target Name=` on a single line.

The following lines should be indented another 2 spaces, so that they're indented *more* than the `<Copy/>` line below it.